Dockerfile

```diff
@@ -8,16 +8,14 @@ LABEL org.label-schema.schema-version="1.0" \
 
 MAINTAINER AlleoTech <admin@alleo.tech>
 
-ARG PHP_VERSION=72
+ARG PHP_VERSION=80
 
 # Enable Networking
 RUN echo "NETWORKING=yes" > /etc/sysconfig/network
 
 # Install EPEL & REMI
-RUN rpm -Uvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm \
-    && rpm -Uvh http://rpms.remirepo.net/enterprise/remi-release-7.rpm \
-    && yum-config-manager --enable epel \
-    && yum-config-manager --enable remi-php${PHP_VERSION}
+RUN yum-config-manager --enable epel \
+  && yum-config-manager --enable remi-php${PHP_VERSION}
```

Comment:
```
Preparing...                          ########################################
	package epel-release-7-14.noarch is already installed
The command '/bin/sh -c rpm -Uvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm     && rpm -Uvh http://rpms.remirepo.net/enterprise/remi-release-7.rpm     && yum-config-manager --enable epel     && yum-config-manager --enable remi-php${PHP_VERSION}' returned a non-zero code: 1
```

17 and 18 lines are causing the above error while building therefore I removed them and it works just fine. Can I proceed with the same changes for PHP 8.1?

Reply:
Looks good to me.

Reply:
OK, thanks!